template.yaml

```diff
@@ -21,8 +23,12 @@ Parameters:
 Mappings:
   Environments:
     dev:
+      ### Where are these ids coming from again? can we import them instead of hardcoding them? Ignore me if this is the same thing
+      ### that I pushed on a while back and we werent able to solve 😅
```

Comment:
The ids you were concerned about before were the elastic ip addresses used to talk to the public internet. I think most of our lambdas hardcode the subnet ids and the VPC. I just looked at the cloudformation templates. The subnet ids come from the `vpc-dev` stack and are in the output of that stack so we could import those. The security group comes from the `platform-services-us-west-2-dev` and is _not_ output from that stack. If the security group was in the output we could remove the environment mapping entirely. Something we should definitely strive for.